src/hotspot/share/logging/logTagSet.hpp

```diff
@@ -165,7 +165,7 @@ class LogTagSetMapping : public AllStatic {
 // will instantiate the LogTagSetMapping template, which in turn creates the static field for that
 // tagset. This _tagset contains the configuration for those tags.
 template <LogTagType T0, LogTagType T1, LogTagType T2, LogTagType T3, LogTagType T4, LogTagType GuardTag>
-LogTagSet LogTagSetMapping<T0, T1, T2, T3, T4, GuardTag>::_tagset(&LogPrefix<T0, T1, T2, T3, T4>::prefix, T0, T1, T2, T3, T4);
+LogTagSet LogTagSetMapping<T0, T1, T2, T3, T4, GuardTag>::_tagset{&LogPrefix<T0, T1, T2, T3, T4>::prefix, T0, T1, T2, T3, T4};
```

Comment:
Is this a specialized initialization syntax? I'm not familiar with its significance.

Reply:
This particular change I'm not very attached to, if it doesn't immediately improve the code to you then I suggest we skip it. This syntax is called "uniform initialization" syntax, I believe. Semantically it is *almost* equivalent to using `()` for initialization. Here I am only using it to make it very clear that we're initializing a static field for a specific class. I think it reads easier at the top-level since with parens it's syntactically similar to a strange function declaration.

Here's a good link: https://stackoverflow.com/questions/24953658/what-are-the-differences-between-c-like-constructor-and-uniform-initialization

Regarding your other comment: This line is what instantiates every single `LogTagSet` which is needed for the logging, and none are created at run time.